models/prices/prices_stability_rank.sql

```diff
@@ -0,0 +1,82 @@
+{{ config(
+        schema='prices',
+        alias = 'stablibity_rank',
+        materialized='table',
+        file_format = 'delta',
+        tags = ['static'],
+        post_hook = '{{ expose_spells(\'["ethereum", "arbitrum", "gnosis", "optimism", "bnb", "avalanche_c", "polygon", "fantom", "celo", "base", "zksync", "zora"]\',
+                                    "sector",
+                                    "prices",
+                                    \'["hosuke"]\') }}'
```

Comment:
lets remove the post-hook, i don't think it needs displayed on data explorer. this is more helpful for backend engineering.